platform-sdk/swirlds-common/src/main/java/com/swirlds/common/wiring/schedulers/internal/SequentialTask.java

```diff
@@ -36,6 +36,20 @@ class SequentialTask extends AbstractTask {
     private final FractionalTimer busyTimer;
     private final UncaughtExceptionHandler uncaughtExceptionHandler;
 
+    /**
+     * If true, then this task will be executed even if scheduler is currently squelching.
+     * <p>
+     * This flag is necessary for flushing, since we want the flush task to execute even if the task scheduler is
+     * squelching normal handling. It isn't known at the time of construction whether a given task will be responsible
+     * for flushing, so we need a dynamic flag that can be set when the flush work is submitted.
+     */
+    private boolean overrideSquelch = false;
```

Comment:
Something you can consider in order to avoid this ugly override is to change sequential tasks to flush in the same way as concurrent schedulers, i.e. by calling `ObjectCounter.waitUntilEmpty()`. It's less efficient, but we don't need to be super efficient when flushing since we only flush at startup and reconnect.

Reply:
Done, for both sequential and sequential thread schedulers